packages/trader/src/Assets/Trading/Categories/trade-categories.jsx

```diff
@@ -422,6 +422,32 @@ const TradeCategories = ({ category }) => {
                     </React.Fragment>
                 );
                 break;
+            case 'vanilla':
+                TradeTypeTemplate = (
+                    <React.Fragment>
+                        <p>
```

Comment:
I suggest to use `Text` component here. 
we have done the same thing for Accumulators.

Reply:
Will make the change